plugins/modules/ipam_subnet.py

```diff
@@ -2436,6 +2436,18 @@ def __init__(self, *args, **kwargs):
         exclude = ["state", "csp_url", "api_key", "id"]
         self._payload_params = {k: v for k, v in self.params.items() if v is not None and k not in exclude}
         self._payload = Subnet.from_dict(self._payload_params)
+        # Safely remove unwanted attributes
+        if self._payload.dhcp_config:
+            self._payload.dhcp_config.abandoned_reclaim_time = None
+            self._payload.dhcp_config.abandoned_reclaim_time_v6 = None
+            self._payload.dhcp_config.echo_client_id = None
+
+            # Handle inheritance sources safely
```

Comment:
```suggestion
            # Unset unsupported DHCP configuration attributes in the inherited DHCP config
```

Reply:
This piece of code was redundant, i have removed it

Reply:
Why is this removed ?
This code was to handle inheritance sources , right ?

Reply:
As discussed on call, its a backend bug and needs all field to be added as part of inheritance